src/datanode/src/heartbeat/handler/downgrade_region.rs

```diff
@@ -43,7 +42,7 @@ impl HandlerContext {
                 Err(err) => InstructionReply::DowngradeRegion(DowngradeRegionReply {
                     last_entry_id: None,
                     exists: true,
-                    error: Some(err.output_msg()),
```

Comment:
Do we have other places use `output_msg()`?

Reply:
> Do we have other places use `output_msg()`?

Some places are by design.🥹